pkg/utils/k8sutil_test.go

```diff
@@ -1106,6 +1106,15 @@ func TestGetProvisionContainer(t *testing.T) {
 		t.Errorf("Unexpected command: %s", c.Args[0])
 	}
 
+	// If the content type is url it should use curl
+	c, err = getProvisionContainer("https://raw.githubusercontent.com/test/test/test/test.py", "sha256:abc1234", "", "test.foo", "url", "python2.7", "unzip", rvol, dvol, lr)
+	if err != nil {
+		t.Errorf("Unexpected error: %s", err)
+	}
+	if !strings.HasPrefix(c.Args[0], "curl https://raw.githubusercontent.com/test/test/test/test.py --silent --output /tmp/func.fromurl") {
+		t.Errorf("Unexpected command: %s", c.Args[0])
+	}
+
```

Comment:
can you add another unit test that checks that if the type is `url+zip` it downloads and unzip the file?

Reply:
Hi @andresmgot, 
Can you add a .zip file that can be used for integration testing to ```examples/nodejs/```? My local integration tests use https://github.com/tkeech1/kubelessfunction/blob/master/nodejs/helloFunctions.zip but I would like to update the tests to download the zip file from the kubeless repo to avoid creating a dependency on my repo. Thanks.

Reply:
It's okay to have it there for the moment, just upload the file with this PR. We can modify the URL after this PR gets merged.